lib/ts/browser.ts

```diff
@@ -0,0 +1,16 @@
+import { application } from './application';
+import * as Stacks from './index';
+
+declare global {
+    interface Window {
+        Stacks: any;
+    }
+}
+
+window.Stacks = {
+    application: application,
+    _initializing: application.isInitialized(),
+    ...Stacks,
+};
```

Comment:
You can [configure webpack to output a UMD module](https://webpack.js.org/configuration/output/#outputlibrary) instead of directly talking to `window` here